src/admin.rs

```diff
@@ -76,6 +81,32 @@ pub fn load_object_store_from_env(
     }
 }
 
+pub async fn run_garbage_collection(
+    path: &Path,
+    object_store: Arc<dyn ObjectStore>,
+    collect_manifests: Option<Duration>,
+    collect_wal: Option<Duration>,
+    collect_compacted: Option<Duration>,
+) -> Result<Arc<DbStats>, Box<dyn Error>> {
+    let opts = GarbageCollectorOptions {
+        manifest_options: collect_manifests.map(|min_age| GarbageCollectorDirectoryOptions {
```

Comment:
This comment still applies in the new PR (somewhat - the config is now resolved in the CLI instead of the admin). I feel like it's a little misleading to run something like:
```
schedule-garbage-collection --manifest min_age=20s,period=10s
```
And then it suddenly starts cleaning up WALs as well, so I left it to not use the default.